examples/angular/src/app/pages/pages.component.ts

```diff
@@ -36,5 +53,9 @@ export class DotCMSPagesComponent implements OnInit {
       .subscribe((data) => {
         this.context.set(data['context']);
       });
+
+    onFetchPageAssetFromUVE((pageAsset) => {
+      this.context.update((context) => ({ ...context, pageAsset }));
```

Comment:
Same as in react, is there dupe data here?

Reply:
Here no, here we override the current pageAsset with the pageAsset sent from UVE